features/user_can_log_out.feature

```diff
@@ -0,0 +1,17 @@
+Feature: User can log out
+    As a User
+    In order to exit my account
+    I would like to be able to log out
+
+    Scenario: User logs out
+        Given the following user exists
+            |name       |  email            | password  |
+            |Zane       |  zane@gmail.com   | paswoord  |
+
+        Given I am logged in as "Zane"
```

Comment:
This scenario seems odd, please refactor this with a background steps and make sure you read 
https://cucumber.io/docs/gherkin/reference/ for some guidance

Reply:
OK. Added background step.